data/acs/01_download.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import pandas as pd
+import json
+
+# Modified from EDM's db-acs/01_download.py file
+# Thank you, @NYCPlanning/EDM :)
+def get_tract(group):
+    '''
+    Downloads requested tract-level table for the five
+    NYC counties and combines into a single table
+    Parameters
+    ----------
+    group: str
+    Returns
+    -------
+    pd DataFrame
+        Contains data from requested ACS 5-year table,
+        all five NYC counties included
+    '''
+    frames = []
+    # Download tract-level tables using census API, and combine into a single NYC DataFrame
+    for county in ['081', '085', '005', '047', '061']:
+        url = f'https://api.census.gov/data/2018/acs/acs5?get=group({group})&for=tract:*&in=state:36&in=county:{county}'
+        resp = requests.request('GET', url).content
+        df = pd.DataFrame(json.loads(resp)[1:])
+        df.columns = json.loads(resp)[0]
+        frames.append(df)
+    return pd.concat(frames)
+
+if __name__ == "__main__":
+    # get tract-level data for all attributes in the B08301 "Means of Transportation" table
+    # For attribute info, see
+    # https://api.census.gov/data/2018/acs/acs5/groups/B08301.html
+    print("Downloading transportation census data...")
+    transportationData = get_tract('B08301')
+    
+
+    # get tract-level data for all attributes in the B01003 "Population" table
+    # For attribute info, see 
+    # https://api.census.gov/data/2018/acs/acs5/groups/B01003.html
+    print("Downloading population census data...")
+    populationData = get_tract('B01003')
+
+    print("Transforming data...")
+
+    # Add 'variable' column to transportation table, to match transportation and final output tables
+    transportationData['variable'] = pd.Series('to be replaced', index=transportationData.index)
```

Comment:
Does pandas throw any warnings when you index using this syntax?

Reply:
I get this warning, but I'm not sure if it's due to `.loc` or `[]`. Will investigate
![image](https://user-images.githubusercontent.com/3311663/74267048-52d93b00-4cd3-11ea-9248-4fcf22ed6562.png)

Reply:
@SPTKL maybe you'd have thoughts on this?

Reply:
what is this for? 
you can just do 
`transportationData['variable'] = transportationData.index`

Reply:
I needed to add the `variable`  column with fluff data to the transportationData table to match the schema of another dataframe and support later using `.loc[]` to transfer its values to that target dataframe (line 58)

Reply:
Previously I saw examples of simply creating the column by assigning it a random value, like `transportationData['variable'] = 'lorem ipsum'`, but that threw some error about indexing, if I remember correctly. Researched online and found this method of adding a new column:
https://stackoverflow.com/a/12555510